operations/system_admin_toolkit/usage/SAT_Bootprep.md

```diff
@@ -486,6 +510,7 @@ input file support rendering as a Jinja2 template and thus support variables:
   `session_templates` key:
     - `name`
     - `configuration`
+    - `bos_parameters.bootsets.rootfs_provider_passthrough`
```

Comment:
The key is `boot_sets` not `bootsets`. Also there is technically another key, which is the name of the boot set between `boot_sets` and `root_fs_provider_passthrough`. Let's try this:

```suggestion
    - `bos_parameters.boot_sets.BOOT_SET_NAME.rootfs_provider_passthrough`
```

Hopefully that will make it clear to the reader.